tcms/xmlrpc/api/bug.py

```diff
@@ -140,9 +140,15 @@ def report(execution_id, tracker_id):
 
     execution = TestExecution.objects.get(pk=execution_id)
     bug_system = BugSystem.objects.get(pk=tracker_id)
+
     if bug_system.base_url:
         tracker = IssueTrackerType.from_name(bug_system.tracker_type)(bug_system)
-        url = tracker.report_issue_from_testcase(execution)
+        url = ""
+        try:
+            url = tracker.report_issue_from_testcase(execution)
+        except:
+            message = "[ERR] Cannot report issue to tracker {}".format(tracker)
+            warnings.warn(message)
```

Comment:
Please explain why we need this change ?

Reply:
Just for debug purpose, will remove it